Update Organization Permissions Documentation (#297)
hashicorp/terraform-docs-common: website/docs/cloud-docs/users-teams-organizations/permissions.mdx

The following organization permissions are available:

### Project permissions

You must select a level of access for projects.

> #### No access

Reviewer: Should this be the "None" that is reflected on the setting page? All the other names seem to correspond directly to the name in the UI

Author: I initially had this heading as "None" for consistency between the UI and docs but upon rereading it, felt like "None" wasn't a very descriptive heading, even though it's nested under the org permissions section. This being said though, I definitely don't feel strongly about either heading and do like keeping things consistent, so I'm happy to defer to y'all! What do you, @emlanctot? 🤔

Reviewer: I don't feel strongly, I think I would keep it consistent to the UI headers. But I would defer to education team !

The following organization permissions are available:

### Project permissions

You must select a level of access for projects.

#### None

Does not grant members any explicit access to projects and workspaces. You can still grant permissions to individual projects or workspaces through [Project Permissions](/terraform/cloud-docs/users-teams-organizations/permissions#project-permissions) and [Workspace Permissions](/terraform/cloud-docs/users-teams-organizations/permissions#workspace-permissions).

#### View all projects

Allows members to view all projects within the organization. This lets users:
- View project names in a given organization.

#### Manage all projects

> Allows members to create and manage all projects and workspaces within the organization. In addition to all the permissions granted by “Manage all workspaces”, this also lets users:

Author: Great suggestion!

- Manage other teams' access to all projects.
- Create, edit, and delete projects (otherwise only available to organization owners).
- Move workspaces between projects.

### Workspace permissions

You must select a level of access for workspaces.

#### None

> Does not grant members any explicit access to projects and workspaces. You can still grant permissions to individual projects or workspaces through [Project Permissions](/terraform/cloud-docs/users-teams-organizations/permissions#project-permissions) and [Workspace Permissions](/terraform/cloud-docs/users-teams-organizations/permissions#workspace-permissions).

Reviewer 2: ```suggestion
Members do not have access to projects or workspaces. You can grant permissions to individual projects or workspaces through [Project Permissions](/terraform/cloud-docs/users-teams-organizations/permissions#project-permissions) or [Workspace Permissions](/terraform/cloud-docs/users-teams-organizations/permissions#workspace-permissions).
```

#### View all workspaces

> Allows members to view all workspaces within the organization. This lets users:

Reviewer 2: ```suggestion
Members can view all workspaces within the organization. This lets users:
```

- View information and features relevant to each workspaces (e.g. runs, state versions, variables).

#### Manage all workspaces

> Allows members to create and manage all workspaces within the organization. This lets users:

Reviewer 2: ```suggestion
Members can create and manage all workspaces within the organization. This lets users:
```

> - Manage all workspaces. Any action that requires admin access is also available to teams with permission to manage workspaces.

Reviewer 2: ```suggestion
- Perform any action that requires admin permissions in those workspaces.
```

> - Create new workspaces (otherwise only available to organization owners) that belong to the "Default Project".

Reviewer 2: ```suggestion
- Create new workspaces within the organization's **Default Project**, an action that is otherwise only available to organization owners. 
```
